Reading List/Reading List/View Controllers/BookDetailViewController.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  BookDetailViewController.swift
+//  Reading List
+//
+//  Created by Norlan Tibanear on 6/6/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+class BookDetailViewController: UIViewController {
+    
+    // Outlets
+    @IBOutlet var titleTextField: UITextField!
+    @IBOutlet var descriptionTextView: UITextView!
+    
+    var bookController: BookController?
+    var book: Book?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        descriptionTextView.layer.borderWidth = 0.5
+        descriptionTextView.layer.borderColor = UIColor.lightGray.cgColor
+        descriptionTextView.clipsToBounds = true
+        
+        updateView()
+    }
+    
+    func updateView() {
+        if let book = book {
+            titleTextField.text = book.title
+            descriptionTextView.text = book.reasonToRead
+            title = book.title
+        } else {
+            title = "Add a new book"
+        }
+        
+    }
+    
+
+    @IBAction func saveBtn_TouchUpInside(_ sender: UIBarButtonItem) {
+        
+        guard let title = titleTextField.text,
+            let description = descriptionTextView.text,
+            title != "" && description != "" else { navigationController?.popViewController(animated: true);return }
```

Comment:
This might be confusing to the user if they don't enter a description and hit save but nothing seems to happen. Maybe consider disabling the save button until they have the required information entered.